index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="kr">
+  <head>
+    <meta charset="UTF-8" />
+    <title>👩🏻‍💻 유튜브 강의실</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="./src/css/index.css" />
+    <script type="module" src="./src/js/index.js"></script>
```

Comment:
script태그를 defer 어트리뷰트 없이 이곳에서 사용하는건 위험해보여요!
```suggestion
    <script defer type="module" src="./src/js/index.js"></script>
```
이 방법 또는 body태그의 바로 위로 옮겨주는게 어떨까요~?